Fluid.Tests/Fluid.Tests.csproj

```diff
@@ -7,12 +7,12 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Microsoft.Extensions.FileProviders.Abstractions" Version="1.1.1" />
```

Comment:
It's a low version on purpose. Unless there is a security issue, extensions should use the smallest possible version such that users don't have to bump their versions. In the end the one from the dotnet runtime will be used.